src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,111 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static int DEFAULT_SIZE = 10;
+    private int size;
+    private T[] elements = (T[]) new Object[10];
+
+    private void ensureCapacity(int minCapacity) {
```

Comment:
1. private methods should be located after public methods
2. this name better explains what does this method do
3. we dont need any parameters for that
```suggestion
        private void growArray() {
```